azure/package.json

```diff
@@ -27,8 +44,32 @@
     "postinstall:lerna:win32": "npm run pre-lerna-install:win32 && (lerna bootstrap --strict --hoist & npm run post-lerna-install:win32)",
     "pre-lerna-install:default": "[ -f package-lock.json ] && mv package-lock.json package-lock.json.bak; [ -f lerna-package-lock.json ] && mv lerna-package-lock.json package-lock.json; exit 0",
     "pre-lerna-install:win32": "if exist package-lock.json move package-lock.json package-lock.json.bak && if exist lerna-package-lock.json move lerna-package-lock.json package-lock.json",
-    "test": "lerna run test --stream --no-bail --no-sort",
-    "tsc": "tsc"
+    "test": "npm run test:mocha && npm run test:jest && npm run test:types && npm run test:realsvc",
+    "test:bail": "npm run test:mocha:bail && npm run test:jest:bail",
+    "test:copyresults": "copyfiles --exclude \"**/node_modules/**\" \"**/nyc/**\" nyc",
+    "test:coverage": "c8 npm run test:report",
+    "test:jest": "assign-test-ports && lerna run test:jest --concurrency 4 --stream --no-bail --no-sort -- -- --color",
+    "test:jest:bail": "assign-test-ports && lerna run test:jest --concurrency 4 --stream",
+    "test:jest:report": "assign-test-ports && lerna run test:jest --concurrency 4 --stream --no-bail --no-sort -- -- --ci --reporters=default --reporters=jest-junit",
+    "test:mocha": "lerna run test:mocha --stream --no-bail --no-sort -- -- --color",
+    "test:mocha:bail": "lerna run test:mocha --stream",
+    "test:mocha:report": "lerna run test:mocha --stream --no-bail --no-sort -- -- --timeout 4s --reporter xunit --reporter-option output=nyc/mocha-junit-report.xml",
+    "test:realsvc": "lerna run test:realsvc --stream --no-bail --no-sort",
+    "test:realsvc:report": "lerna run test:realsvc:report --stream --no-bail --no-sort",
+    "test:report": "npm run test:mocha:report && npm run test:jest:report && npm run test:realsvc:report",
+    "test:types": "lerna run test:types --stream --no-bail --no-sort",
+    "tsc": "lerna run tsc --stream",
+    "tsc:fast": "fluid-build  --root .. -s tsc",
+    "tsfmt": "lerna run tsfmt:fix --no-sort --stream --no-bail",
+    "tsfmt:fix": "lerna run tsfmt:fix --no-sort --stream --no-bail",
+    "typetests:gen": "fluid-type-validator -m packages -v",
+    "typetests:prepare": "npm run typetests:gen -- -p",
+    "watch": "concurrently \"npm run watch:tsc\" \"npm run watch:esnext\" \"npm run watch:webpack\"",
+    "watch:esnext": "lerna run --parallel build:esnext -- -- --watch",
+    "watch:tsc": "lerna run --parallel tsc -- -- --watch",
+    "watch:webpack": "lerna run --parallel webpack -- -- --watch",
+    "webpack": "lerna run --no-sort webpack --stream",
+    "webpack:profile": "lerna run --no-sort webpack:profile --stream"
```

Comment:
I'm guessing all of these scripts will run as is in the azure folder too?

Reply:
yup